### ISSUE TITLE
Exception when posting to API on Linux

### ISSUE THREAD

**Issue by glitchybunny** [Facepunch/sbox-issues#1526]
**Describe the bug**
Upon launching the game, it immediately encounters the following errors. No gamemodes or addons can be fetched from the API, making the game unplayable:
```
Exception when posting to API
There was a problem retrieving account information
...
Api Error: "https://apix.facepunch.com/sandbox-asset?id=facepunch.sandbox"
No package found for facepunch.sandbox
```

**To Reproduce**

1. This occurs consistently on Proton Experimental (5.15.7-1-MANJARO). It also affects proton version 7.0.1 and older on my system.

**Expected behavior**
The API loads correctly and the game behaves normally.

**Screenshots**
![image](https://user-images.githubusercontent.com/12066089/150024997-14734559-02c9-450a-a0de-ed834797a360.png)
![image](https://user-images.githubusercontent.com/12066089/150025047-79f20471-d19b-4557-9122-466ed261b87e.png)

**Additional context**
I've tried ruling out different network configurations, using both WiFi and 5G, allowing or blocking the firewall, etc. The API site works correctly in the browser, so I don't think it's a network issue on my end. Loading https://apix.facepunch.com/sandbox-asset?id=facepunch.sandbox returns the expected value:
```
{"Asset":{"Org":{"Ident":"facepunch","Title":"Facepunch","Description":"","Thumb":"https://files.facepunch.com/sbox/org/facepunch/logo.1.png","SocialTwitter":"","SocialWeb":""},"Ident":"sandbox","Title":"Sandbox","Summary":"Create and play in your own sandbox mode.","Description":"This game lets you spawn props and ragdolls and create entities to play with. It\u0027s like Garry\u0027s Mod.\r\n\r\nThis is a bit of a sandbox itself. This is not the final sandbox gamemode. All of this code will likely be thrown away and re-written when we come to do the sandbox gamemode proper.","Thumb":"https://files.facepunch.com/sbox/org/facepunch/sandbox/thumb.860fd8a7.jpg","Background":"https://files.facepunch.com/sbox/org/facepunch/sandbox/background.b5f12342.jpg","PackageType":2,"Updated":1618500112,"Download":{"Type":"github","Url":"https://github.com/Facepunch/sandbox"},"ConfigJson":"{\u0022MinPlayers\u0022:1,\u0022MaxPlayers\u0022:32,\u0022NetworkType\u0022:0,\u0022MapList\u0022:[\u0022facepunch.construct\u0022],\u0022MapSelection\u0022:0,\u0022RankType\u0022:0}","UsersNow":4,"UsersDay":130,"UsersMonth":1293,"UsersTotal":7428,"Tags":["multiplayer","mousekeyboard","singleplayer","coop"],"CategoryId":1,"SubCategoryId":0,"Favourited":279,"VotesUp":304,"VotesDown":14}}
```

**Comment by garrynewman** [comment]
That 0x80090304 error is going to be the culprit. Do you know whether you got that in windows or linux?

**Comment by darkrilin** [comment]
The 0x80090304 error is occurring on linux. Windows seems to be working fine now.

**Comment by Cameron-Stuart** [comment]
If you use `protontricks` to `winecfg` your prefix, you can change the Windows version from Windows 10 to Windows 7 which resolved the issue with the main menu for me. However the 0x80090304 error is still stopping SSL from working, preventing the ability to download new gamemodes.

**Comment by handsomematt** [comment]
This was fixed with Proton 7.0 or something
